test/test_ops.py

```diff
@@ -982,6 +983,16 @@ def test_complex_half_reference_testing(self, device, dtype, op):
                 *transformed_sample.args,
                 **transformed_sample.kwargs,
             )
+            # Since range of chalf is much less compared to cfloat,
+            # we get `inf`s easily (eg. with `pow`, `exp`),
+            # so we cast `cfloat` back to `chalf`.
```

Comment:
cc: @mruberry (we discussed this offline previously)

Reply:
Casting cfloat back to chalf is fine, but probably we should also modify inputs to hit overflows less frequently, because overflows mask real issues.

Reply:
And despite this you still need to skip?

Reply:
Skips are for the test in BinaryUfuncs which doesn't cast back.

Reply:
As per offline discussion with @mruberry , will file an issue that we have a low quality test for fast growing ops like `pow` and we can discuss ways to improve it independently later.